internal/server/db/cluster/update.go

```diff
@@ -106,6 +106,20 @@ var updates = map[int]schema.Update{
 	67: updateFromV66,
 	68: updateFromV67,
 	69: updateFromV68,
+	70: updateFromV69,
+}
+
+// updateFromV69 adds description column to certificate.
+func updateFromV69(ctx context.Context, tx *sql.Tx) error {
+	q := `
+	ALTER TABLE certificates ADD COLUMN description TEXT NOT NULL;
```

Comment:
This is going to fail to run on any Incus system that has a DB entry in the certificates table.

This is going to need a `DEFAULT ""` to avoid this problem.

Reply:
ahh, that's right. Fixed.